burrow/assets/service_checks.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "agent_version": "5.1.0",
```

Comment:
Is this the agent version you are using? Asking because there are libs available on newer versions that can reduce the implementation of the check

Reply:
Adjusted, i'm using 6.18.1